back/spec/factories/custom_fields.rb

```diff
@@ -235,7 +235,7 @@
       input_type { 'page' }
       page_layout { 'default' }
 
-      factory :custom_field_form_end_page do
```

Comment:
Minor and not blocking, but I would leave as `custom_field_form_end_page` as the key is `form_end` so feels more specific

Reply:
Sure, I'll change it to `custom_field_form_end_page`.